netebpfext/net_ebpf_ext.c

```diff
@@ -651,6 +749,20 @@ net_ebpf_ext_register_providers()
         return STATUS_UNSUCCESSFUL;
     }
 
+    return_value = ebpf_ext_attach_register_provider(
```

Comment:
Are you missing an input argument? This should be catched at compile time

Reply:
The input arguments are there (next line)!

Reply:
I'm saying ebpf_ext_attach_register_provider() requires 4 input parameters but you only have 3 here. Does this even compile?

Reply:
![image](https://user-images.githubusercontent.com/88061203/128250266-55dc1d7d-14b7-46fc-8d5a-2ba8f877efe5.png)
It does compile. I think there is only 3 inputs.